util/configuration.py

```diff
@@ -125,7 +127,7 @@ class Email:
     SECTION_NAME = "Email"
     __data = File.data.get("Email", {})
     address = __data.get("address", "<your.email@gmail.com>")
-    password = __data.get("password", "<your password>")
+    password = __data.get("password", "<your password>") # TODO: Encrypt
```

Comment:
Why is there a TODO here?

Reply:
Is there still another step that needs done?

Reply:
Yeah, I forgot to delete that TODO comment. I made it because I was searching through the config to see if there were multiple passwords that would need updated, but it was just the one. I will remove that comment with my next push.